rules/windows/credential_access_lsass_openprocess_api.toml

```diff
@@ -137,6 +137,7 @@ api where host.os.type == "windows" and
         "?:\\Program Files\\Cisco\\AMP\\*\\sfc.exe",
         "?:\\Program Files\\Common Files\\McAfee\\AVSolution\\mcshield.exe",
         "?:\\Program Files\\EA\\AC\\EAAntiCheat.GameService.exe",
+        "?:\\Program Files\\Elastic\\Agent\\data\\elastic-agent-*\\components\\agentbeat.exe",
```

Comment:
@joe-desimone can you update the updated date when you get a chance before merging.

Reply:
done